drivers/i2s/i2s_common.c

```diff
@@ -16,7 +16,7 @@ int z_impl_i2s_buf_read(const struct device *dev, void *buf, size_t *size)
 	ret = i2s_read((const struct device *)dev, &mem_block, size);
 
 	if (!ret) {
-		struct i2s_config *rx_cfg =
+		const struct i2s_config *rx_cfg =
```

Comment:
I suggest to make it uniform for all rx_cfg/tx_cfg: 
```c
const struct i2s_config *rx_cfg;

rx_cfg = i2s_config_get((const struct device *)dev, I2S_DIR_RX);
```

Reply:
Done.